### PR TITLE
Messing with indenting again.

### PR DESCRIPTION
**Pull request by jlconlin**
There were some blank lines missing which caused the indenting to get messed up. Also, in some places there was too much indenting and it didn't capture that it was code.

I discovered this when I turned it into a web page and saw that it was awful.Take a look now. It should look correct.

### CODE REVIEW

**Comment by apmccartney** [comment]
This looks like a jeckyl bug. You've compiled locally to confirm the proposed changes resolve the issue?

File: Style/Code.md [hunk 1]
```diff
@@ -93,60 +99,60 @@ for (...; ...; ...) {
...
 1. Prefer to snuggle type sigil to the base type rather than the variable name in variable declarations, e.g.
 
-    ```cpp
-    T* p; /* good */
-    T& p; /* good */
-    T *p; /* bad */
-    T* p, q; /* OOPS put these on separate lines */
-    ```
+```cpp
+T* p; /* good */
```
Comment (apmccartney) [inline Style/Code.md]: This is an example of 7.1
It probably ought to be at the same indentation level

File: Style/Code.md [hunk 2]
```diff
@@ -93,60 +99,60 @@ for (...; ...; ...) {
...
 1. Prefer to access non-static class and struct member variables and methods through the `this` pointer, e.g.
 
-    ```cpp
-    double
-    MyClass::method( int i ) const {
-      const auto foo = this->otherMethod( i );
-      return foo * this->fieldVariable;
-    }
-    ```
+```cpp
```
Comment (apmccartney) [inline Style/Code.md]: This is an example of 6.3
It probably ought to be at the same indentation level

**Comment by jlconlin** [comment]
There was an extra space before the closing triple tick. That was causing some issues. 

FYI: The code has to be indented to the same level as the bulleted list. For example:

1. Blah 1

2. Blah 2

    ^ Stuff has to be indented this far to "belong" to Blah 2. Note that there has to be a blank line.
3. Blah 3

Here is some stuff that isn't indented. Note how it resets the counter.

4. Blah 4

**Comment by apmccartney** [comment]
approved (I can't merge)

**Pull request merged**